Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -188,34 +212,65 @@ public static string GetJsonReportPath(string jsonPath)
 			return string.Empty;
 		}
 
-		public static void CreateDirectory(string path)
+		public void CreateDirectory(string path)
 		{
 			if (!Directory.Exists(path))
 			{
 				Directory.CreateDirectory(path);
 			}
 		}
 
-		private static string GetInstalledStudioShortVersion()
+		public Dictionary<string, bool> AddToDictionary(BindingList<LanguageDetails> languages)
+		{
+			var dictionaryResult = new Dictionary<string, bool>();
+			if (languages != null)
+			{
+				foreach (var item in languages)
+				{
+					dictionaryResult.Add(item.LanguageName, item.IsChecked);
```

Comment:
This will throw an ArgumentException if an element with the same key already exists in the dictionary.

Reply:
I've added validation to add item only when dictionary doesn't already have one with the same key